notebooks/mean_var_calcs.ipynb

```diff
@@ -32,6 +32,7 @@
    "source": [
```

Comment:
<p>this is where I try to load areacella, and if it doesn't work, it gets a none flag.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/apendergrass/cmip6hack-precip/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>